habit-backend/users/admin.py

```diff
@@ -0,0 +1,40 @@
+from .forms import UserAdminCreationForm, UserAdminChangeForm
+from django.contrib import admin
+from django.contrib.auth.models import Group
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from django.contrib.auth.forms import ReadOnlyPasswordHashField
+
+from users.models import User
+
+class UserAdmin(BaseUserAdmin):
+    # The forms to add and change user instances
+    form = UserAdminChangeForm
+    add_form = UserAdminCreationForm
+
+    # The fields to be used in displaying the User model.
+    # These override the definitions on the base UserAdmin
+    # that reference specific fields on auth.User.
+    list_display = ('email', 'is_superuser')
```

Comment:
Let's also display `first_name`; `last_name` in the admin view?

Reply:
Sure, will do that!